PrimaryInstallScript.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+if [ $EUID -ne 0 ]; then
+	echo "Please run as root user"
+	exit 1
+fi
+
+ping -q -c1 google.com > /dev/null
+ 
+if [ $? -ne 0 ]; then
+	echo "No Internet Connection...Aborting"
+	exit 1
+fi
+
+#install Epsilon-Hermes
+git clone https://github.com/UCSolarCarTeam/Epsilon-Hermes.git /home/pi/Epsilon-Hermes/
+(cd /home/pi/Epsilon-Hermes && cd ../ && mv ./Epsilon-Hermes ./src \
+	&& mkdir Epsilon-Hermes \
+	&& mv ./src ./Epsilon-Hermes/)
+qmake /home/pi/Epsilon-Hermes/src/
+make /home/pi/Epsilon-Hermes/src/
+mv /home/pi/build/Epsilon-Hermes /opt/
+
+git clone https://github.com/alanxz/SimpleAmqpClient /tmp/SimpleAmqpClient/
```

Comment:
The SimpleAmqp stuff should go before the Epsilon-Hermes install, hermes might not work.